accounting/accounting/doctype/stock_entry/stock_entry.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2023, Akhil and contributors
+// For license information, please see license.txt
+
+// frappe.ui.form.on("Stock Entry", {
+// 	refresh(frm) {
+
```

Comment:
There should be some client side code to compute computed fields like total/amount on client side.